include/crud-terms.php

```diff
@@ -159,28 +159,32 @@ public function save_term( $term_id, $tt_id, $taxonomy ) {
 	 *
 	 * @param string[] $taxonomies Queried taxonomies.
 	 * @param array    $args       WP_Term_Query arguments.
-	 * @return PLL_Language|string|false The language(s) to use in the filter, false otherwise.
+	 * @return PLL_Language[] The languages to use in the filter.
 	 */
-	protected function get_queried_language( $taxonomies, $args ) {
+	protected function get_queried_languages( $taxonomies, $args ): array {
 		global $pagenow;
 
-		// Does nothing except on taxonomies which are filterable
-		// Since WP 4.7, make sure not to filter wp_get_object_terms()
+		/*
+		 * Do nothing except on taxonomies which are filterable.
+		 * Since WP 4.7, make sure not to filter wp_get_object_terms().
+		 */
 		if ( ! $this->model->is_translated_taxonomy( $taxonomies ) || ! empty( $args['object_ids'] ) ) {
-			return false;
+			return array();
 		}
 
-		// If get_terms is queried with a 'lang' parameter
+		// If get_terms() is queried with a 'lang' parameter.
 		if ( isset( $args['lang'] ) ) {
-			return $args['lang'];
+			$languages = is_string( $args['lang'] ) ? explode( ',', $args['lang'] ) : $args['lang'];
+			$languages = is_array( $languages ) ? $languages : array( $languages );
+			return array_filter( array_map( array( $this->model, 'get_language' ), $languages ) );
```

Comment:
This means `$args['lang']` can only be a string, an integer, or an array. Am I right?

Do you think we should trim the slugs?

Also, what do you think of this?

```suggestion
			$languages = is_string( $args['lang'] ) ? explode( ',', $args['lang'] ) : $args['lang'];
			return array_filter( array_map( array( $this->model, 'get_language' ), (array) $languages ) );
```